MutationObserver.js

```diff
@@ -464,7 +501,7 @@ window.MutationObserver = window.MutationObserver || (function(undefined) {
                      */
                     elestruct.attr = reduce($target.attributes, function(memo, attr) {
                         if (!config.afilter || config.afilter[attr.name]) {
-                            memo[attr.name] = attr.value;
+                            memo[attr.name] = getAttributeValue($target, attr);
```

Comment:
Resolving a mutation will be questionable because we don't actually know what the value of the element is. We'll end up providing all `cssText` I'm assuming